### ISSUE TITLE
Unclear description

### ISSUE THREAD

**Issue by sindresorhus** [Quramy/electron-disclosure#1]
> It captures your desktop by 5 minutes and posts tweet with the captured images. (Don't worry, post tweets requires your authentication)

What does this mean? Does it capture a screenshot every 5 minutes and tweets it? If so, why?

**Comment by Quramy** [comment]
Thanks for comment. I apologize that it has confuse you.

This app is a sample application and is useless...

I've wanted to make something to tell the following functions:
- How to capture desktop image in an Electron app
- How to connect to resources controlled by a OAuth in an Electron app

I think you open this issue because my PR.
And, If you want your "Example apps" list keeps to includes useful apps, you don't have to merge my PR.

**Comment by sindresorhus** [comment]
Yes. I think the format might be better as a blog post describing the approach and having this repo as something the user can try out afterwards. Just a repo with some code isn't that useful.

The intention of the repo could also be clearer.

**Comment by Quramy** [comment]
OK. I will append to the README the intention that I made my repo.

And also thanks for other issues and commit's comments.
